assets/build/install.sh

```diff
@@ -49,15 +49,19 @@ fi
 #       are both installed.
 PG_GEM=$(grep 'gem "pg"' ${REDMINE_INSTALL_DIR}/Gemfile | awk '{gsub(/^[ \t]+|[ \t]+$/,""); print;}')
 MYSQL2_GEM=$(grep 'gem "mysql2"' ${REDMINE_INSTALL_DIR}/Gemfile | awk '{gsub(/^[ \t]+|[ \t]+$/,""); print;}')
+SQLITE3_GEM=$(grep -A 1 'gem "sqlite3"' ${REDMINE_INSTALL_DIR}/Gemfile | awk '{gsub(/^[ \t]+|[ \t]+$/,""); print;}')
```

Comment:
https://github.com/redmine/redmine/blob/master/Gemfile#L67

I just happened to look at the master branch and noticed this will break probably on next release.  My version should cover this.